src/solvers/refinement/string_constraint_generator_main.cpp

```diff
@@ -17,42 +17,48 @@ Author: Romain Brenguier, romain.brenguier@diffblue.com
 #include <util/pointer_predicates.h>
 #include <util/ssa_expr.h>
 
+unsigned string_constraint_generatort::next_symbol_id=1;
+
```

Comment:
Why is this initialization not done right in the class declaration? I'd rather drop this one instead of the in-class one as you've done in one of your commits.